README.md

```diff
@@ -558,11 +556,12 @@ This is how this publisher works:
 
 1. Read the `version` field from `build.json`
 2. Clone the `gh-pages` branch of the target repository to a local working directory
```

Comment:
May want to hint that the `gh-pages` branch can be configured to something else...

Reply:
Done